src/legacy/ui/public/vis/editors/default/components/default_editor_agg_param_props.ts

```diff
@@ -32,12 +32,13 @@ import { SubAggParamsProp } from './default_editor_agg_params';
 export interface AggParamCommonProps<T> {
   agg: AggConfig;
   aggParam: AggParam;
+  disabled?: boolean;
   editorConfig: EditorConfig;
   indexedFields?: FieldParamType[];
   showValidation: boolean;
   state: VisState;
   value: T;
-  responseValueAggs: AggConfig[] | null;
+  metricAggs: AggConfig[] | null;
```

Comment:
According to the changes `metricAggs` isn't set to `null`:
```suggestion
  metricAggs: AggConfig[];
```